### PR TITLE
Grafana Live telemetry streaming

### PR DESCRIPTION
**Pull request by apenza**
# Description
Created application for streaming telemetry data live to Grafana for faster updates vs. reading from database.
Also created application to automate setup of Grafana Live streaming.

# How Has This Been Tested?
Manual testing on both local and remote Grafana instances.

# Checklist:
- [x] New functionality is documented in the necessary spots (i.e new functions documented in the header)
- [x] Unit tests cover any new functionality or edge cases that the PR was meant to resolve (if applicable) 
- [x] The CI checks are passing
- [x] I reviewed my own code in the GitHub diff and am sure that each change is intentional

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,3 +38,32 @@ Stop the service with:
...
+The service account can have any display name but must be given the Admin role ([More information about creating service accounts](https://grafana.com/docs/grafana/latest/administration/service-accounts/)).
+3. Click "Add service account token" and then "Generate token". The token can have any display name. Don't set an expiration date unless you want the token to become invalid after that date. 
+Copy the token to the clipboard. If you lose it, you will need to generate a new token.
+4. Set the environment variable `GRAFANA_LIVE_TOKEN` to the service account token from the previous step. This can be done by creating a file called `.env` in the root directory.
+The file contents should be `GRAFANA_LIVE_TOKEN="<token>"`, replacing `<token>` with the service account token.
+
+Live data is not currently set up for radio module.
+
+### Configuration
+By default, the application is configured using the file `grafana_live.yaml` in the `data/config` directory. If the flag `-c (FILE_NAME)` is used, the application will instead parse the configuration file at `data/config/(FILE_NAME).yaml`.
```
Comment (AarC10) [inline README.md:60]: Maybe move the radio mod comment below this?

File: cmd/grafana_live/live_setup.go [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+		return
+	}
+
+	// set .env file to contain token (or print token)
+	fmt.Println("Adding service account token to .env file...")
+	setEnvFile(scanner, key)
+
+	// create the Grafana Live dashboard from JSON
+	dashboardURL := grafanaURL.JoinPath("dashboards", "db")
+	err = createDashboard(dashboardURL, "data/grafana/Backplane-Live.json")
```
Comment (AarC10) [inline cmd/grafana_live/live_setup.go:244]: Too hardcoded

**Review by AarC10 — APPROVED** [review]
Awesome! Just see my 2 comments. Well done

**Comment by AarC10** [comment]
Can we actually merge main though.

**Review by Smartinez3324 — APPROVED** [review]
Looks good 🔥
